turbo.json

```diff
@@ -17,6 +19,12 @@
     },
     "clean": {
       "cache": false
+    },
+    "topo": {
+      "dependsOn": ["^topo"]
+    },
+    "typecheck": {
+      "dependsOn": ["topo"]
```

Comment:
https://turbo.build/repo/docs/core-concepts/monorepos/task-dependencies#dependencies-outside-of-a-task

Adding `topo` allows `typecheck` tasks in `formspree-core` and `formspree-react` to run in parallel while ensuring that changing in `formspree-core` will cause a cache miss in `formspree-react`.

An alternative is to do: `"typecheck": { "dependsOn": ["^typecheck"] }` but it'll cause `formspree-react` to always run after `formspree-core`.